examples/basic_query/example_client.py

```diff
@@ -34,37 +18,45 @@ def predict(host, uid, x):
     print("'%s', %f ms" % (r.text, latency))
 
 
-def add_mnist_app(host):
+def add_example_app(host):
```

Comment:
Yeah that's a good idea. I was thinking about trying to avoid a dependency on the clipper_manager to keep things simple for the example, but the example is illustrating how to query Clipper not how to initialize a model or application.